src/index.mjs

```diff
@@ -71,17 +70,18 @@ function isIgnored(node, filter) {
  * @param {Array|RegExp|Function} options.ignores - Custom filter(s) that run after origin checks
  * @param {Number} options.timeout - Timeout after which prefetching will occur
  * @param {Function} options.timeoutFn - Custom timeout function
+ * @param {Number} options.limit - Limit the number of links to be observed
  */
 export default function (options) {
   options = Object.assign({
     timeout: 2e3,
     priority: false,
     timeoutFn: requestIdleCallback,
     el: document,
+    limit: Infinity
```

Comment:
I'm OK with `Infinity` being the default for `limit`. A minified version of `Infinity` would be `1/0`, which I want to say `microbundle` should handle for us. cc @lukeed 

An alternative design could have been only accepting a `limit` value if it wasn't undefined, but would probably lead to a longer implem :)

Reply:
Microbundle doesn't do that (yet?)

I'm so proud 🤧